subgraph.template.yaml

```diff
@@ -54,6 +54,8 @@ dataSources:
       abis:
         - name: FixedProductMarketMakerFactory
           file: ./abis/FixedProductMarketMakerFactory.json
+        - name: ConditionalTokens
```

Comment:
why is this newly needed?

Reply:
needed because in `getMarket` in ctf-utils, we need access to the CTF contract to calculate the collectionId:

https://github.com/Polymarket/polymarket-subgraph/blob/7d2d51daab9ef217daf2657e69cd681b2bbaa064/src/utils/ctf-utils.ts#L103